tests/sequence/data/missing_linear_vs_circular.gb

```diff
@@ -0,0 +1,165 @@
+LOCUS       SCU49845     5028 bp    DNA             PLN       21-JUN-1999
```

Comment:
this is actually the sample file supplied by GenBank when describing the gb flat file format. it wasn't working before.